polus-generalized-linear-model-plugin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM r-base:latest
```

Comment:
Make sure to use a specific version of R, not the latest. This will keep consistency in the builds.